### DISCUSSION TITLE
compartir documentos

### DISCUSSION THREAD

**Discussion by clarodri64** [chamilo/chamilo-lms#5400] [General]
En un curso, donde estan inscritos dos profesores, un profesor envia un documento en COMPARTIR DOCUMENTOS a un alumno. Hay forma en que el segundo profesor pueda ver tambien estos archivos entre el primer docente y los alumnos del mismo curso ?
Gracias.

**Comment by NicoDucou** [comment]
Hola @clarodri64 
No existe esta opcion, seria un nuevo desarollo.

Saludos

**Comment by Kaneda-1** [comment]
Hola @clarodri64 

Utiliza la carpeta Documentos del curso para compartir ficheros para todos.

También puedes utilizar enlaces a Google Drive, Dropbox, etc. e insertarlos en Chamilo.
